docs/ReleaseNotes.md

```diff
@@ -1,6 +1,15 @@
 # Acuminator Release Notes
 This document provides information about fixes, enhancements, and key features that are available in Acuminator.
 
+## Acuminator 2.1.2
+Acuminator 2.1.2 includes the bug fixes described in this section, as well as the features that have been implemented in previous versions.
+
+### Fixed Bugs
+In this version of Acuminator, the following bugs have been fixed:
+- Diagnostic suppression was not supported for interfaces, structs, enumerations, and delegates.
```

Comment:
@k-s-popova it wasn't supported only for suppression with suppression file. I think it's better specify this

Reply:
```suggestion
- Diagnostic suppression with suppression file was not supported for interfaces, structs, enumerations, and delegates.
```